doc/deployment.md

```diff
@@ -0,0 +1,94 @@
+﻿# Deploying The Application
+
+This document covers information relating to the current application setup in the dev and production
+environments, and how to deploy new versions of CSLabs.
+
+## Webserver
+
+The application uses a webserver to direct web requests to the correct place. Depending on the requested 
+resource, the webserver will either respond with a static web resource in the case of
+the front-end application, or will direct the request to the back-end application service running on the
+server. The back-end service must be running and responding to requests in order for the front-end to 
+work properly.
+
+Currently, the application uses Nginx as its webserver. The application is hosted on internal virtual
+machines managed via Proxmox on the internal cluster. There is one virtual machine for the dev environment
+and another for production.
+
+## Nginx
+
+Nginx must be installed on the server and configured to direct requests to the appropriate location. The
+configuration file that configured Nginx to do this correctly is committed to the back-end repository, in
+the root directory. `nginx.conf.dev` is the configuration for the dev environment, and `nginx.conf` is the
```

Comment:
Can you name this `nginx.dev.conf`? That will help the editor know it's a conf file.

Reply:
You know what I think I forgot to commit this file anyway 😅

I'll get it committed under `nginx.dev.conf` later tonight.